xAPI.md

```diff
@@ -4998,7 +5004,7 @@ The following table lists xAPI scope values:
 		<td>(re)Define Activities and Actors. If storing a Statement 
 			when this is not granted, ids will be saved and the LRS 
 			MAY save the original Statement for audit purposes, but 
-			SHOULD NOT update its internal representation of any 
```

Comment:
This is a little out of scope of this PR but I brought it in as a was handling a similar change in response to feedback.